ivy/ivy.xml

```diff
@@ -105,6 +105,10 @@
 		<dependency org="org.mortbay.jetty" name="jetty-client" rev="6.1.22" conf="test->default" />
 		<dependency org="org.mortbay.jetty" name="jetty" rev="6.1.22" conf="test->default" />
 		<dependency org="org.mortbay.jetty" name="jetty-util" rev="6.1.22" conf="test->default" />
+		<dependency org="tomcat" name="jasper-runtime" rev="5.5.23" conf="test->default" />
```

Comment:
The tomcat dependencies were previously pulled in by Hadoop 2.4.0. They are needed for the protocol-http[client] JUnit tests using JSP.